packages/graphql-mesh-server/lib/pipeline.ts

```diff
@@ -110,6 +115,49 @@ export class CodePipelineService extends Construct {
       ],
     });
 
+    if (props.cloudFrontDistributionId) {
+      const invalidateCacheLambda = new NodejsFunction(
+        this,
+        "InvalidateCacheLambda",
+        {
+          entry: path.resolve(
+            __dirname,
+            "../assets/handlers/invalidate-cloudfront-cache.ts"
+          ),
+          description: "Lambda function to invalidate CloudFront cache.",
+          runtime: Runtime.NODEJS_18_X,
+          handler: "index.handler",
+          timeout: Duration.seconds(5),
+          environment: {
+            PATHS: "/graphql",
+            DISTRIBUTION_ID: props.cloudFrontDistributionId || "",
```

Comment:
You don't need the `||` here as you are checking for `cloudFrontDistributionId` above

Reply:
Ah yeah, must have added this before I included the if statement